azimuth/config.py

```diff
@@ -450,17 +450,25 @@ def load(cls, config_path: Optional[str], load_config_history: bool) -> "Azimuth
 
         if load_config_history:
             config_history_path = cfg.get_config_history_path()
-            try:
-                with jsonlines.open(config_history_path, mode="r") as config_history:
-                    *_, last_config = config_history
-            except (FileNotFoundError, ValueError):
-                log.info("Empty or invalid config history.")
-            else:
+            last_config = cls.load_last_from_config_history(config_history_path)
+            if last_config:
                 log.info(f"Loading latest config from {config_history_path}.")
-                return AzimuthConfigHistory.parse_obj(last_config).config
+                return last_config
+            else:
+                log.info("Empty or invalid config history.")
```

Comment:
Now that this is a `if`, and no longer a `try`, we can remove the `else`:
```suggestion

            log.info("Empty or invalid config history.")
```

Reply:
Right, because we return in the `if`.